tripy/docs/conf.py

```diff
@@ -150,11 +151,19 @@ def process_docstring(app, what, name, obj, options, lines):
                     pname = "*" + pname
 
                 if pname == "self":
```

Comment:
We can simplify the logic here:
```py
if pname != "self" or obj.__qualname__ in FUNC_W_DOC_VERIF:
    assert (
                pname in documented_args
                ), f"Missing documentation for parameter: '{pname}' in: '{obj}'. Please ensure you've included this in the `Args:` section. Note: Documented parameters were: {documented_args} {doc}"
else:
    assert (
                param.annotation == signature.empty
                ), f"Avoid using type annotations for the `self` parameter since this will corrupt the rendered documentation! Note: Documented parameters were: {documented_args} {doc} {obj.__qualname__} {FUNC_W_DOC_VERIF}"
```